### ISSUE TITLE
BEMHTML: Use replace()

### ISSUE THREAD

**Issue by indutny** [bem/bem-components#1497]
**Blocked by: https://github.com/enb-bem/enb-bemxjst/issues/37**

Please consider using `replace()` instead of:

``` js
block('b1').def()(function() {
  applyCtx({ ... });
});
```

**Comment by tadatuta** [comment]
`extend()` is not working now (see https://github.com/bem/bem-xjst/issues/180) so I change issue title to just `replace()` and create another issue for `extend()`.
